### ISSUE TITLE
Enhance load script files

### ISSUE THREAD

**Issue by Romitou** [SkriptLang/Skript#2951]
### Description
Hello, I'm not sure if this is already implemented, at least I haven't figured out how to do it, but it would be beneficial to add folder support for this effect. Example: `!disable script "games/"` does not unload all the scripts in the file, but will deactivate the folder by adding the prefix -, which will not be taken into account.

![image](https://user-images.githubusercontent.com/35146758/80413234-6f970f00-88cf-11ea-81bb-07ff62340a1e.png)
Current behavor:
![image](https://user-images.githubusercontent.com/35146758/80413311-948b8200-88cf-11ea-9014-f6014be786f6.png)
Purpose of my suggestion:
![image](https://user-images.githubusercontent.com/35146758/80413485-d4526980-88cf-11ea-8461-08f54711dcd3.png)

To summarize, I would like to know if it is possible that the behavior of this event is similar to the commands of Skript (`/sk (dis|en)able <folder>/`). Have a nice day!
